<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartOptionsMain</class>
 <widget class="QDialog" name="StartOptionsMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StartOptionsMain Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QStackedWidget" name="QStackTutorialContainer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QWidget" name="page_3">
      <property name="font">
       <font>
        <italic>false</italic>
       </font>
      </property>
     </widget>
     <widget class="QWidget" name="page_4"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="RestoreWallet">
       <property name="text">
        <string>Restore Wallet</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="NewWallet">
       <property name="text">
        <string>Create New Wallet</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout1">
     <item>
      <widget class="QPushButton" name="Back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
